Source/WarpX.cpp

```diff
@@ -772,16 +763,18 @@ WarpX::ReadParameters ()
 
         // If WarpX::do_dive_cleaning = 1, set also WarpX::do_pml_dive_cleaning = 1
         // (possibly overwritten by users in the input file, see query below)
-        if (do_dive_cleaning)
-        {
-            do_pml_dive_cleaning = 1;
-        }
+        if (do_dive_cleaning) do_pml_dive_cleaning = 1;
+
+        // If WarpX::do_divb_cleaning = 1, set also WarpX::do_pml_divb_cleaning = 1
+        // (possibly overwritten by users in the input file, see query below)
+        // TODO Implement div(B) cleaning in PML with FDTD and remove second if condition
+        if (do_divb_cleaning && maxwell_solver_id == MaxwellSolverAlgo::PSATD) do_pml_divb_cleaning = 1;
```

Comment:
If `do_pml_divb_cleaning` is not set in the inputfile we may warn the user that we are setting it equal to 1 in this case

Reply:
Thanks for the suggestion, Luca! So, if I understand correctly, you would want to warn about a default value of a parameter that is not set by the user? I think we usually don't do that, while we do warn if a parameter set by the user is overwritten automatically. What would be the reason for warning about this default value specifically? I can certainly add the warning (would require to also check that PML is used by looking at the field boundary conditions, because I think the `do_pml` flag is not yet initialized at this point -- it would not make sense to warn about a PML parameter (div cleaning) for a simulation where there is no PML), but I just want to understand the rationale a bit more.